initialize/services.go

```diff
@@ -42,8 +43,39 @@ func (i *Init) Services() error {
 		if err != nil {
 			return fmt.Errorf("failed to initialize libvuln: %v", err)
 		}
+
+		// configure notifier service
+		dInterval, err := time.ParseDuration(i.conf.Notifier.DeliveryInterval)
+		if err != nil {
+			return &clairerror.ErrNotInitialized{
+				Msg: "notifier: failed to parse delivery interval: " + err.Error(),
+			}
+		}
+		pInterval, err := time.ParseDuration(i.conf.Notifier.PollInterval)
+		if err != nil {
+			return &clairerror.ErrNotInitialized{
+				Msg: "notifier: failed to parse poll interval: " + err.Error(),
+			}
+		}
+
```

Comment:
Ah, I see, the config and the notifier opts have some differing types.

Why doesn't the config just have the needed types? The yaml package handles Durations correctly: https://play.golang.org/p/MyhYUozoQBo

Reply:
Just following the current pattern of things.